packages/contracts/contracts/tutorials/EtherDelta.sol

```diff
@@ -0,0 +1,168 @@
+pragma solidity ^0.4.19;
+
+import "../utils/SafeMath.sol";
+import "./AccountLevels.sol";
```

Comment:
How about putting all EtherDelta contracts in a sub-folder? Just so it's clear what is and is not part of it.